runtime/calamari/src/lib.rs

```diff
@@ -222,7 +222,7 @@ impl pallet_balances::Config for Runtime {
 	type Event = Event;
 	type ExistentialDeposit = NativeTokenExistentialDeposit;
 	type AccountStore = frame_system::Pallet<Runtime>;
-	type WeightInfo = pallet_balances::weights::SubstrateWeight<Runtime>;
+	type WeightInfo = weights::pallet_balances::SubstrateWeight<Runtime>;
```

Comment:
@stechu @Dengjianping 
 `SubstrateWeight` is the new old name of struct which implements the `WeightInfo` trait. That's what our new template file outputs.